x-pack/plugins/enterprise_search/common/types/app_search.ts

```diff
@@ -23,3 +23,42 @@ export interface IRole {
     availableRoleTypes: string[];
   };
 }
+
+export interface IApiToken {
```

Comment:
These are in the common types directory, because I will be using them in the upcoming Credentials View PR.